modules/tinymce/src/core/test/ts/webdriver/paste/CopyAndPasteTest.ts

```diff
@@ -0,0 +1,172 @@
+import { RealClipboard, RealMouse } from '@ephox/agar';
+import { describe, it } from '@ephox/bedrock-client';
+import { TinyAssertions, TinyHooks, TinySelections, TinyUiActions } from '@ephox/wrap-mcagar';
+
+import Editor from 'tinymce/core/api/Editor';
+
+describe('webdriver.tinymce.core.paste.CopyAndPasteTest', () => {
+  const hook = TinyHooks.bddSetup<Editor>({
+    base_url: '/project/tinymce/js/tinymce',
+    toolbar: false,
+    statusbar: false
+  }, []);
+
+  interface Selection { startPath: number[]; soffset: number; finishPath: number[]; foffset: number }
```

Comment:
This type should be at the top of the file... Also this should not be needed as you should be able to use the `CursorPath` type instead.

- [ ] Use the `CursorPath` type instead of a duplicate/custom type

Reply:
done